zerver/tests/test_push_notifications.py

```diff
@@ -367,12 +375,14 @@ def test_send_notification_endpoint(self) -> None:
             logger.output,
             [
                 "INFO:zilencer.views:"
-                f"Sending mobile push notifications for remote user 6cde5f7a-1f7e-4978-9716-49f69ebfc9fe:{hamlet.id}: "
+                f"Sending mobile push notifications for remote user 6cde5f7a-1f7e-4978-9716-49f69ebfc9fe:id:{hamlet.id}: "
```

Comment:
These test changes should be in the previous commit, where the behaviour changed, so we keep bisectability.

Reply:
Rebased and re-pushed to split the test changes up.